django/contrib/admin/widgets.py

```diff
@@ -362,21 +373,28 @@ def id_for_label(self, id_):
 
 
 class AdminTextareaWidget(forms.Textarea):
+    use_fieldset = True
```

Comment:
You've added `use_fieldset = True` onto widgets that won't want to use a fieldset